content/docs/start/data-pipelines.md

```diff
@@ -92,19 +92,19 @@ prepare:
 - `-o data/prepared` specifies an output directory for this script, which writes
   two files in it. This is how the <abbr>workspace</abbr> should look like now:
 
-  ```diff
-      .
-      ├── data
-      │   ├── data.xml
-      │   ├── data.xml.dvc
-  +   │   └── prepared
-  +   │       ├── test.tsv
-  +   │       └── train.tsv
-  +   ├── dvc.yaml
-  +   ├── dvc.lock
-      ├── params.yaml
-      └── src
-          ├── ...
+  ```
+   .
+   ├── data
+   │   ├── data.xml
+   │   ├── data.xml.dvc
+  +│   └── prepared
+  +│       ├── test.tsv
+  +│       └── train.tsv
+  +├── dvc.yaml
+  +├── dvc.lock
+   ├── params.yaml
+   └── src
+       ├── ...
   ```
```

Comment:
OK so this is what `git diff` would output. Unfortunately it's still not being rendered nicely by our engine, which eats up the single trailing spaces used for indentation 🙁 

![image](https://user-images.githubusercontent.com/1477535/93161647-95834b80-f6d8-11ea-8dfd-e01df84f83bb.png)

Reply:
This is only a problem in this block because the whole block is indented inside an MD bullet.

So we're back to square one 😒 should we manually align this? Or look for a solution in https://github.com/iterative/dvc.org/issues/1761#issuecomment-692439599?